rasa/core/training/story_reader/yaml_story_reader.py

```diff
@@ -25,154 +28,202 @@
 
 
 class YAMLStoryReader(StoryReader):
-    """Class that reads the core training data in a YAML format"""
+    """Class that reads Core training data and rule data in YAML format"""
```

Comment:
actually I think the prior version was already general enough, no?

Reply:
yes it was but I wanted to change `in a YAML` to `in YAML`. Are you saying I should remove the `and rule data` part again?

Reply:
>  Are you saying I should remove the and rule data part again?

Yep. `Core training data` sums it up nicely, doesn't it?

Reply:
👍 updated